experiments/run_experiment.py

```diff
@@ -39,10 +46,8 @@
 import pandas as pd
 
 
-_RESULTS_DIR = (  # MRMC/experiment_results/mrmc_results
-    pathlib.Path(os.path.abspath(__file__)).parent.parent.parent
-    / "experiment_results/mrmc_results"
-)
+# MRMC/experiment_results/mrmc_results
+_RESULTS_DIR = _MRMC_PATH / "experiment_results"
```

Comment:
Is the purpose of using pathlib library only to be able to use this '/' path joining? If so, I think it'd be lighter weight to us os.path.join for these operations, which reduces the necessary imports and is fairly canonical (and explicit) for Python I/O.

Reply:
Near the head of the file, pathlib is used to create the _MRMC_PATH. It lets us construct the absolute path to the MRMC install location from the filepath of the current file without relying on path.split('/'), which seems less robust.